openjdk8/README.md

```diff
@@ -38,15 +38,30 @@ The entry point for the openjdk8 image is [docker-entrypoint.bash](https://githu
 
 If the default command (java) is used, then the entry point sources the [setup-env.bash](https://github.com/GoogleCloudPlatform/appengine-java-vm-runtime/blob/master/openjdk8/src/main/docker/setup-env.bash), which looks for supported features to be enabled and/or configured.  The following table indicates the environment variables that may be used to enable/disable features, any default values if they are not set and the environment variable that holds the resulting JVM arguments that actually enable the feature: 
 
-| Feature              | directory    | Enable/Disable  | Default | JVM args      |
-|----------------------|--------------|-----------------|---------|---------------|
-| Stackdriver Debugger | /opt/cdbg/   | $DBG_ENABLE     | true    | $DBG_AGENT    |
-| Temporary file       |              | $TMPDIR         |         | $SET_TMP      |
-| Java options         |              | $JAVA_OPTS      |         | $JAVA_OPTS    |
+| Feature              | directory  | Enable/Disable  | Default            | JVM args           |
+|----------------------|------------|-----------------|--------------------|--------------------|
+| Stackdriver Debugger | /opt/cdbg/ | $DBG_ENABLE     | true               | $DBG_AGENT         |
+| Temporary file       |            | $TMPDIR         |                    | $JAVA_TMP_OPTS     |
+| Temporary file       |            | $HEAD_SIZE      | from /proc/meminfo | $JAVA_HEAP_OPTS    |
```

Comment:
It is actually a graph of env variable dependencies and users can set any of them.
I try an alternate format that shows that better.
stand by